index.html

```diff
@@ -0,0 +1,1249 @@
+                        " class="iframe-container" sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </div>
+
+            </section>
+            <!-- Buttons -->
+
+            <!-- Responsive image -->
+
+            <section id="image">
+                <div class="code-container">
+                    <h2 class="component-title">Responsive Image</h2>
+                    <img class="img-responsive" src="https://source.unsplash.com/random" />
+                    <iframe
+                        src="https://carbon.now.sh/embed?bg=rgba%2874%2C118%2C159%2C1%29&t=seti&wt=none&l=auto&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cimg%2520class%253D%2522img-responsive%2522%2520src%253D%2522https%253A%252F%252Fsource.unsplash.com%252Frandom%2522%2520%252F%253E"
+                        " class="iframe-container" sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </div>
+
+                <div class="code-container">
+                    <h2>Rounded Image</h2>
+                    <img class="img-responsive rounded" src="https://source.unsplash.com/random" />

```

Comment:
Round image is also not responsive it has a fixed width.

Reply:
fixed